lib/storage-client.js

```diff
@@ -296,9 +297,9 @@ class ChromedriverStorageClient {
       }
     }
 
-    if (_.isString(minBrowserVersion) && !Number.isNaN(minBrowserVersion)) {
+    const minBrowserVersionInt = convertToInt(minBrowserVersion);
+    if (minBrowserVersionInt) {
```

Comment:
do we also need to reject zeroes?

Reply:
I think zero should be valid technically.  let me fix it to validate with null.

Reply:
fixed https://github.com/appium/appium-chromedriver/pull/291/commits/b90aecc025e87d9310f1adc8b6f2b05fdf1173dc